src/main/java/uk/gov/companieshouse/web/pps/service/penaltypayment/PayablePenaltyPaymentService.java

```diff
@@ -1,10 +1,10 @@
-package uk.gov.companieshouse.web.lfp.service.latefilingpenalty;
+package uk.gov.companieshouse.web.pps.service.penaltypayment;
 
 import uk.gov.companieshouse.api.model.latefilingpenalty.PayableLateFilingPenalty;
 import uk.gov.companieshouse.api.model.latefilingpenalty.PayableLateFilingPenaltySession;
-import uk.gov.companieshouse.web.lfp.exception.ServiceException;
+import uk.gov.companieshouse.web.pps.exception.ServiceException;
 
-public interface PayableLateFilingPenaltyService {
+public interface PayablePenaltyPaymentService {
```

Comment:
Already mentioned, not sure if we rename this PayablePenaltyService and remove the Payment bit. Any input is appreciated.

Reply:
What about 'PenaltyPaymentService', and remove the 'Payable'? Or even just 'PaymentService' as they are all penalties of some sort.

Reply:
Currently changed to PayablePenaltyService, happy to discuss about that.